### ISSUE TITLE
Turbo_stream format not being sent anymore

### ISSUE THREAD

**Issue by colmexdev** [hotwired/turbo-rails#106]
Greetings!

   I'm currently building a web app on Rails with Turbo and Stimulus. It works great. However, Irecently upgraded gem versions and, all of a sudden, turbo_stream format (that was sent by default on visit and form submission) stopped working; therefore, some of my endpoints can't render response.

   My issue here is: Was it all due to the upgrading? Or what can be happening that stopped turbo_stream format from being returned.

   I'm working on Ruby 3.0.0, Rails 6.1.1 and Turbo-rails 0.5.9.

   Thanks in advance for all your help.

**Comment by en30** [comment]
Did you upgrade `@hotwired/turbo` npm package also?
The Turbo Streams MIME type has changed since `@hotwired/turbo@v7.0.0-beta.3`, 
from `text/html; turbo-stream` to `text/vnd.turbo-stream.html`.
https://github.com/hotwired/turbo/releases
So I suspect content negotiation is not working as expected because of version mismatch.

**Comment by colmexdev** [comment]
Thank you very much! I didn't read the release notes so I never thought on rebuilding the npm package. Now it's working like a charm.

   Thank you very much!!! 🎉🎉

**Comment by jakehockey10** [comment]
I've ran yarn upgrade and it says I'm using these versions now:

``` shell
├─ @hotwired/turbo-rails@7.0.0-beta.5
├─ @hotwired/turbo@7.0.0-beta.4
```

but the POST request is still of html format.  Is there something else I need to do?

**Comment by bolom** [comment]
I have same issue did you find a solution @jakehockey10 ?

**Comment by jakehockey10** [comment]
No I did not.  I keep restarting the same tutorial over and over, and always get this issue at the same spot.  I gave up a while ago.  I'm sorry I can't be of any help.  But I wish you good luck!

**Comment by CharlieIGG** [comment]
For anyone stumbling upon this:

As noted above, Turbo changed its standard for the ContentType from `text/html; turbo-stream` to` text/vnd.turbo-stream.html`. Depending on the version that you have of both the NPM package, and the Ruby gem, you might be having a mis-match in the format. If you're using `form_with ... format: :turbo_stream`, this relies on the right MIME::Type from being set in the initializers, [which was changed after version 0.5.1 and again after 0.5.4](https://github.com/hotwired/turbo-rails/commit/495eb1e34463db9ace802eee8b5338ef6e178e43#diff-a6c9ea9b91a99345ff04b9597b41369bc2552724c7c00135db376ff782aaf7abR39).

TL;DR - Make sure your running the latest stable version of both the NPM package and the Ruby gem.
